app/views/SidebarView/index.tsx

```diff
@@ -127,17 +138,14 @@ class Sidebar extends Component {
 		let isAdmin = false;
 
 		if (roles) {
-			isAdmin = allPermissions.reduce((result, permission) => {
-				if (permission) {
-					return result || permission.some(r => roles.indexOf(r) !== -1);
-				}
-				return result;
-			}, false);
+			// Creating a new array with booleans values, than the every check if it is a truthy array
+			isAdmin = allPermissions.map(permission => permission.some(r => roles.indexOf(r) !== -1)).every(Boolean);
```

Comment:
when we don't have admin permissions we get this error.


![Screen Shot 2021-11-02 at 00 15 15](https://user-images.githubusercontent.com/14894380/139782925-063209c5-b475-4996-902c-1ea0b83267d0.png)


the const `allPermissions` is like this: `[
  undefined,
  undefined,
  undefined,
  undefined
]`

Reply:
Nice, thanks, I didn't realize this!